bazel/grpc_build_system.bzl

```diff
@@ -57,24 +57,17 @@ def _maybe_update_cc_library_hdrs(hdrs):
       ret.append(h)
   return ret
 
-def _maybe_update_cc_library_defines(name):
-  ret = []
-  if name == "alts_proto":
```

Comment:
double checking, the generated output no longer needs `PB_FIELD_16BIT=1` ?

Reply:
I think so. The only reason we defined PB_FIELD_16BIT=1 there is to hope it could pass it down to nanopb source compilation, which obviously did not work out.